.github/workflows/release-node.yml

```diff
@@ -55,33 +52,44 @@ jobs:
           retry_wait_seconds: 60 # wait between retries
           command: cd ${{ matrix.image-name }} && npm i --dry-run
       - 
-        name: Build image with node version and latest tag
+        name: Build image with latest tag
+        uses: docker/build-push-action@v2
+        if: ${{ env.RELEASE_TAG == 'latest' && matrix.node-version == env.NODE_LATEST }}
+        with:
+          context: ./${{ matrix.image-name }}
+          file: ./${{ matrix.image-name }}/Dockerfile
+          build-args: NODE_VERSION=${{ matrix.node-version }}
+          load: true
+          tags: apify/actor-${{ matrix.image-name }}:${{ env.RELEASE_TAG }}
+      - 
+        name: Build image with node version tag
         uses: docker/build-push-action@v2
         if: ${{ env.RELEASE_TAG == 'latest' }}
         with:
           context: ./${{ matrix.image-name }}
           file: ./${{ matrix.image-name }}/Dockerfile
-          build-args: NODE_VERSION=${{ env.NODE_VERSION }}
+          build-args: NODE_VERSION=${{ matrix.node-version }}
           load: true
-          tags: apify/actor-${{ matrix.image-name }}:${{ env.RELEASE_TAG }},apify/actor-${{ matrix.image-name }}:${{ env.NODE_VERSION }}
+          tags: apify/actor-${{ matrix.image-name }}:${{ matrix.node-version }}
       -
         name: Build image with release tag
         uses: docker/build-push-action@v2
-        if: ${{ env.RELEASE_TAG != 'latest' }}
+        if: ${{ env.RELEASE_TAG != 'latest' && matrix.node-version == env.NODE_LATEST }}
         with:
           context: ./${{ matrix.image-name }}
           file: ./${{ matrix.image-name }}/Dockerfile
-          build-args: NODE_VERSION=${{ env.NODE_VERSION }}
+          build-args: NODE_VERSION=${{ matrix.node-version }}
           load: true
-          tags: apify/actor-${{ matrix.image-name }}:${{ env.RELEASE_TAG }}
+          tags: apify/actor-${{ matrix.image-name }}:${{env.RELEASE_TAG}}-${{ env.RELEASE_TAG }}
```

Comment:
should be `matrix.node-version` no? This would create `:beta-beta` tag or something.